### PR TITLE
[FIX] Remove doubled "Canned Responses" strings

### PR DESCRIPTION
**Pull request by matheusbsilva137**
## Proposed changes (including videos or screenshots)

 - Remove doubled canned response setting introduced in #22703 (by setting id change);
 - Update "Canned Responses" keys to "Canned_Responses".

## Issue(s)

[Task - ClickUp](https://app.clickup.com/t/7084cm)

## Steps to test or reproduce

Go to Administration and search for 'Canned Responses'. Only **one** setting (menu) should be displayed in the sidebar.

### CODE REVIEW

**Review by pierre-lehnen-rc — DISMISSED** [review]

**Review by ggazzo — CHANGES_REQUESTED** [review]
Could we check and mention where this duplication was introduced? and btw I think we have some bad translations "Canned Responses" instead "Canned_Responses", could you use this issue and fix together?

**Comment by matheusbsilva137** [comment]
> Could we check and mention where this duplication was introduced? and btw I think we have some bad translations "Canned Responses" instead "Canned_Responses", could you use this issue and fix together?

I've just updated all the "Canned Responses" keys to "Canned_Responses" (keeping the _best_ translations for each language).
This duplication has been introduced in #22703 (in the `ee/app/canned-responses/server/settings.js` file changes).

File: packages/rocketchat-i18n/i18n/es.i18n.json [hunk 1]
```diff
@@ -710,6 +709,7 @@
   "Canned_Response_Sharing_Private_Description": "Solo usted y los administradores de Livechat pueden acceder a esta respuesta predefinida",
   "Canned_Response_Sharing_Public_Description": "Cualquiera puede acceder a esta respuesta predefinida",
   "Create_your_First_Canned_Response": "Crea tu primera respuesta predefinida",
+  "Canned_Responses": "Respuesta predefinida",
```
Comment (KevLehman) [inline packages/rocketchat-i18n/i18n/es.i18n.json:712]:
```suggestion
  "Canned_Responses": "Respuestas predefinidas",
```

**Review by KevLehman — COMMENTED** [review]
Thanks for fixing this :) (there's a conflict on the migration btw)

**Review by tassoevan — APPROVED** [review]

**Review by tassoevan — APPROVED** [review]

**Pull request merged**